docroot/themes/humsci/humsci_basic/humsci_basic.theme

```diff
@@ -8,6 +8,26 @@
 use Drupal\Component\Utility\Html;
 use Drupal\node\Entity\Node;
 
+/**
```

Comment:
ack!! I forgot about theme settings!! Brilliant. Could we include one note about where this code came from? (in the su_humsci theme)

can we also clarify the comment on lines 22 about the specfic way that only developer permissions are allowed to change the block title, so allowing the description to show is the workaround?